### ISSUE TITLE
C is so strange.

### ISSUE THREAD

**Issue by waternoob1005** [OpenFarLands/TheStoneProxy#11]
```
2024/06/15 19:14:58 Сlient connected: 127.0.0.1:64326
2024/06/15 19:15:19 Client disconnected: 127.0.0.1:64326
```
![Uploading 화면 캡처 2024-06-15 191651.png…]()

**Comment by waternoob1005** [comment]
![image](https://github.com/OpenFarLands/TheStoneProxy/assets/137138524/03e241a9-675c-475e-9969-09a431f8c06f)

**Comment by mikhaillav** [comment]
Yep, it was Russian `C`. Thank you!

**Comment by mikhaillav** [comment]
Fixed in [v3.0.1 release](https://github.com/OpenFarLands/TheStoneProxy/releases/tag/v3.0.1)
